pkg/cmd/status/status.go

```diff
@@ -0,0 +1,616 @@
+	if err != nil {
+		return err
+	}
+
+	var events []Event
+	var resp []Event
+	pages := 2
+	for page := 1; page <= pages; page++ {
+		query.Add("page", fmt.Sprintf("%d", page))
+		p := fmt.Sprintf("users/%s/received_events?%s", currentUsername, query.Encode())
+		err := c.REST(ghinstance.Default(), "GET", p, nil, &resp)
+		if err != nil {
+			var httpErr api.HTTPError
+			if !errors.As(err, &httpErr) || httpErr.StatusCode != 404 {
+				return fmt.Errorf("could not get events: %w", err)
+			}
+		}
+		events = append(events, resp...)
+		if len(resp) == 0 || len(resp) < perPage {
+			break
+		}
+	}
+
+	s.RepoActivity = []StatusItem{}
+
+	for _, e := range events {
+		if s.Org != "" && e.Org.Login != s.Org {
+			continue
+		}
+		if s.ShouldExclude(e.Repo.Name) {
+			continue
+		}
+		si := StatusItem{}
+		var number int
+		switch e.Type {
+		case "IssuesEvent":
+			if e.Payload.Action != "opened" {
+				continue
+			}
+			si.Reason = "new issue"
+			si.preview = e.Payload.Issue.Title
+			number = e.Payload.Issue.Number
+		case "PullRequestEvent":
+			if e.Payload.Action != "opened" {
+				continue
+			}
+			si.Reason = "new PR"
+			si.preview = e.Payload.PullRequest.Title
+			number = e.Payload.PullRequest.Number
+		case "PullRequestReviewCommentEvent":
+			si.Reason = "comment on " + e.Payload.PullRequest.Title
+			si.preview = e.Payload.Comment.Body
+			number = e.Payload.PullRequest.Number
+		case "IssueCommentEvent":
+			si.Reason = "comment on " + e.Payload.Issue.Title
+			si.preview = e.Payload.Comment.Body
+			number = e.Payload.Issue.Number
+		default:
+			continue
+		}
+		si.Repository = e.Repo.Name
+		si.Identifier = fmt.Sprintf("%s#%d", e.Repo.Name, number)
+		s.RepoActivity = append(s.RepoActivity, si)
+	}
+
+	return nil
+}
+
+func statusRun(opts *StatusOptions) error {
+	client, err := opts.HttpClient()
+	if err != nil {
+		return fmt.Errorf("could not create client: %w", err)
+	}
+
+	sg := NewStatusGetter(client, opts)
+	errc := make(chan error)
+
+	// TODO break out sections into individual subcommands
+
+	opts.IO.StartProgressIndicator()
+	go func() {
+		err := sg.LoadNotifications()
+		if err != nil {
+			err = fmt.Errorf("could not load notifications: %w", err)
+		}
+		errc <- err
+	}()
+
+	go func() {
+		err := sg.LoadEvents()
+		if err != nil {
+			err = fmt.Errorf("could not load events: %w", err)
+		}
+		errc <- err
+	}()
+
+	go func() {
+		err := sg.LoadSearchResults()
+		if err != nil {
+			err = fmt.Errorf("failed to search: %w", err)
+		}
+		errc <- err
+	}()
+
+	for i := 0; i < 3; i++ {
+		if err := <-errc; err != nil {
+			return err
+		}
+	}
+	opts.IO.StopProgressIndicator()
+
+	cs := opts.IO.ColorScheme()
+	out := opts.IO.Out
+	fullWidth := opts.IO.TerminalWidth()
+	halfWidth := (fullWidth / 2) - 2
+
+	idStyle := cs.Cyan
+	leftHalfStyle := lipgloss.NewStyle().Width(halfWidth).Padding(0).BorderRight(true).BorderStyle(lipgloss.NormalBorder())
+	rightHalfStyle := lipgloss.NewStyle().Width(halfWidth).Padding(0)
```

Comment:
Would we consider adding 1-character padding around the vertical border between the halves?
<img width="139" alt="image" src="https://user-images.githubusercontent.com/887/160456367-bf0206a7-b80e-4282-8525-98ff685e0c3d.png">

I could go either way, but I think the padding might add readability.

Reply:
I like the idea of adding some padding here 👍

Reply:
It was there to begin with but I got greedy about using all available width ;_; I can put it back.